packages/drivers/odsp-driver/src/fetchSnapshot.ts

```diff
@@ -88,6 +88,12 @@ export async function fetchSnapshotWithRedeem(
     removeEntries: () => Promise<void>,
     enableRedeemFallback?: boolean,
 ): Promise<ISnapshotContents> {
+    // For back. compatibility purposes. To be removed with #8784.
```

Comment:
Please use the following format:
// back-compat: <some text, including version when added OR can be removed>